core/src/main/resources/error/error-classes.json

```diff
@@ -5680,4 +5687,4 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
This line generated by `SparkThrowableSuite`.

Reply:
After https://github.com/apache/spark/pull/41256, the empty line should be generated at the end. cc @panbingkun . Just in case, @beliefer have you rebased on the recent master?